### PR TITLE
More power nullability

### PR DESCRIPTION
**Pull request by collinlunn**

### CODE REVIEW

File: Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#nullable enable
+using System.Diagnostics.CodeAnalysis;
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs:2]: Doesn't seem used?

File: Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/LightBulbComponent.cs [hunk 2]
```diff
@@ -52,7 +53,7 @@ public class LightBulbComponent : Component, ILand, IBreakAct
             set
             {
                 _color = value;
-                OnLightColorChange?.Invoke(this, null);
+                OnLightColorChange?.Invoke(this, new EventArgs());
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/LightBulbComponent.cs:56]: Does this work if you do EventHandle<EventArgs?>?

Comment (collinlunn) [reply Content.Server/GameObjects/Components/Power/ApcNetComponents/PowerReceiverUsers/LightBulbComponent.cs]: worked

File: Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs (hunk 1 above)
Comment (collinlunn) [reply Content.Server/GameObjects/Components/Power/BaseNetConnectorComponent.cs:2]: Used for [NotNullWhen] on a TryGet method

**Pull request merged**